<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWindow</class>
 <widget class="QDockWidget" name="DockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>650</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Dock</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="totalPlayTime">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Steps:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalSteps">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pauseBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:rgb(0, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="solutionBtn">
      <property name="text">
       <string>Solution</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="hintBtn">
      <property name="text">
       <string>Hint</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="newGameBtn">
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetBtn">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Reset Game</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
